### PR TITLE
e2e_inference: Add rhods accelerator profile for Intel flex GPU

### PR DESCRIPTION
**Pull request by vbedida79**
- Latest RHODS 2.4.0 operator includes accelerator profile support
This PR adds-
- RHODS accelerator profile yaml for Intel Data Center Flex 140 GPU added
-  Readme steps to use it with openvino notebook image until new operator version is published
Signed-off-by: vbedida79 [veenadhari.bedida@intel.com](mailto:veenadhari.bedida@intel.com)

### CODE REVIEW

File: e2e/inference/README.md [hunk 1]
```diff
@@ -29,10 +29,26 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
...
+Create AcceleratoProfile in the ```redhat-ods-applications``` namespace 
+
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml```
+
+3. Navigate to ```openvino-notebooks``` ImageStream and add the above created ```AcceleratorProfile``` key to the annotation field, as shown in the image below:
+
+![Alt text](/docs/images/openvino-accelerator-field.png)
+
+4. Access ```rhods-dashboard``` route in the ```redhat-ods-applications``` namespace as in the image below
+
```
Comment (uMartinXu) [inline e2e/inference/README.md:43]: Nativegate to ... in the image blow. Click the location link, the RHODS Dashboard can be launched.

Comment (vbedida79) [reply e2e/inference/README.md:43]: We can actually remove this part, we already mention it here https://github.com/intel/intel-technology-enabling-for-openshift/tree/main/e2e/inference#verification

File: e2e/inference/README.md [hunk 2]
```diff
@@ -29,10 +29,26 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
 ## Work with interactive mode
 To enable the interactive mode, the OpenVINO notebook CR needs to be created and integrated with RHODS.  
 1.	Click on the ```create Notebook``` option in this [link](https://github.com/red-hat-data-services/odh-deployer) from the web console and follow these [steps](https://github.com/openvinotoolkit/operator/blob/main/docs/notebook_in_rhods.md) to create the notebook CR.
-2.	Enable Intel Data Center GPU 
+2.	Enable Intel Data Center GPU - **Technical Preview feature**
```
[the remark this answers is not in the document]
Comment (vbedida79) [reply e2e/inference/README.md]: updated

File: e2e/inference/README.md (hunk 1 above)
Comment (uMartinXu) [reply e2e/inference/README.md:43]: I still prefer to keep it here, since it is not a verification, it is a must-have step to use Intel GPU.

Comment (vbedida79) [reply e2e/inference/README.md:43]: updated

**Review by hershpa — APPROVED** [review]

**Comment by uMartinXu** [comment]
LGTM!

**Pull request merged**